common/logisticspipes/gui/guidebook/GuiGuideBookTabButton.java

```diff
@@ -0,0 +1,52 @@
+package logisticspipes.gui.guidebook;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.ResourceLocation;
+
+import lombok.Setter;
+
+import logisticspipes.LPConstants;
+
+public class GuiGuideBookTabButton extends GuiButton {
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
+
+	public boolean isSet, isActive;
```

Comment:
existing code rarely to never uses definition of multiple properties in one line. Please use existing code style.

Reply:
with an additional note: in the rendering code above, I am fine with it. Although I am not yet sure, if you need that many variables.

Reply:
It eases the thinking but I could try to use less in the future